lib/DDG/Spice/Piratetalk.pm

```diff
@@ -0,0 +1,30 @@
+package DDG::Spice::Piratetalk;
+
+# ABSTRACT: Write an abstract here
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
+spice to => 'http://pirate-speak.herokuapp.com/piratespeak?text=$1';
```

Comment:
It's actually preferred to use HTTP if it doesn't redirect to HTTPS. The client is connected to use via HTTPS so they're fine.